packages/frontend/src/components/common/ThirdPartySignInButton/index.tsx

```diff
@@ -42,7 +42,7 @@ const ThirdPartySignInButtonBase: FC<
                     : ''
             }`}
             leftIcon={
-                <Image width={20} src={logo} alt={`${providerName} logo}`} />
+                <Image width={16} src={logo} alt={`${providerName} logo}`} />
```

Comment:
@IrakliJani any other way I could give a value to an image `width`? 'sm'/'md' don't work :/

Reply:
can you use mantine's Image here?

Reply:
This is mantine's `Image`...

Reply:
Or you mean that I probably shouldn't use it?

Reply:
oops sorry, thought it was a styled.image. can you try `w` instead of width?